asyncstdlib/asynctools.py

```diff
@@ -324,3 +325,55 @@ async def compute_something_else() -> float:
     return __func(
         *[await arg for arg in args], **{k: await arg for k, arg in kwargs.items()}
     )
+
+
+def make_awaitable(function: Callable[..., T]) -> Callable[..., Awaitable[T]]:
```

Comment:
Had to ponder the name a bit. I feel that this shorter one is more suitable:
```suggestion
def sync(function: Callable[..., T]) -> Callable[..., Awaitable[T]]:
```
This matches the naming scheme of relying on ``import asyncstdlib as a`` – so it gives the name ``a.sync`` in practice.